src/_data/resources.json

```diff
@@ -265,6 +275,27 @@
 			"browsers": [
 				"Chrome"
 			]
+		},
+				{
+			"title": "axe",
+			"description": "Chosen by Microsoft, Google, development and testing teams everywhere, axe is the World’s leading digital accessibility toolkit.",
```

Comment:
Can we rephrase this to speak to what axe does, not who uses it? Right now it's unclear.